code/modules/hydroponics/seed_datums.dm

```diff
@@ -1503,6 +1503,7 @@
 	display_name = "gnomes"
 	force_layer = 3
 	chems = list("magicdust" = list(5,20))
+	harvest_sound = 'sound/items/hooh.ogg'
```

Comment:
Should be put in a seed_datums_vr.dm file.

```/datum/seed/gnomes
harvest_sound = 'sound/items/hooh.ogg'```

That's all there has to be listed, since it's the only thing we change.